### DISCUSSION TITLE
🥳 We are now officially in HACS 🥳

### DISCUSSION THREAD

**Discussion by wills106** [wills106/homeassistant-solax-modbus#30] [Announcements]
![image](https://user-images.githubusercontent.com/18155231/155838418-9507813d-d718-45d5-a5db-4145b5364a20.png)
:partying_face: We are now officially in HACS :partying_face:

**Comment by motclem** [comment]
hooray brilliant news :) Thankyou @wills106

**Comment by GpMissi** [comment]
I can't figure out how to install. Searching for the add-on does not appear to me. Can you help me?

**Reply by wills106** [reply]
Do you have HACS installed on your Home Assistant?

HACS is different to the add-on store on HASSIO / Home Assistant OS

https://hacs.xyz/docs/setup/download

You should just need to Click on HACS on the left toolbar / Menu
Then Integrations
Type in solax

Mines showing Update as I already have it installed.

![image](https://user-images.githubusercontent.com/18155231/157720988-b778dc05-3c96-4477-a751-89924cde436e.png)

**Reply by GpMissi** [reply]
I copied the folder to HA and after a reset it finally appeared.

**Reply by wills106** [reply]
Yes if you manualy install you need to restart HA after any changes before they work.
Same with updates, if I release 0.4.8 you need to copy the files over and restart HA before the changes take place.

(Even if you install / update through HACS you still need to restart HA)
